snippets/c.json

```diff
@@ -96,6 +96,15 @@
 		],
 		"description": "Code snippet for #if"
 	},
+	
+	"printf": {
+		"prefix": "pri",
+		"body": [
+			"printf(\"$1\"$2);",
```

Comment:
Why is `$2` needed?

Moreover, can you also do version bump in package.json?

Reply:
The second $2 is for , new cursor placement incase we have any variables , pressing tab will take us there.

For version bump i will refactor it